tests/test_overrides_parser.py

```diff
@@ -1493,19 +1494,19 @@ def error(msg: str) -> Any:
             "interval(1.0, 2.0)",
             CastResults(
                 int=CastResults.error(
-                    "ValueError while evaluating 'int(interval(1.0,2.0))':"
+                    "ValueError while evaluating 'int(interval(1.0, 2.0))':"
```

Comment:
1. these test changes are not about interval.
2. they are not comprehensive.

I think it's better to add a section to function testing that adds whitespace in all sorts of places.
```
foo( )
foo ()
foo(a )
foo( a)
foo(a, b)
foo(a ,b)
```

etc

Reply:
The extra whitespace in the error message is just a side-effect of not throwing whitespace to the hidden channel anymore, so `getText()` still contains them. Obviously these tests are not about testing whitespace handling.

I haven't looked closely at all existing tests, and it's indeed possible that there weren't enough tests about whitespaces before. I only added a few ones in ab52f476 to highlight the failure case of #873, which is what this PR was targeted at. But I'll see about adding more as you suggested.

Reply:
> But I'll see about adding more as you suggested.

I added more in 81390659

Reply:
Oh, I missed the meaning of the change.
Yes, it does make sense.

> (side note: I had to rebase to fix CI tests)

Let me guess, deprecations from cfg.pretty()?

Reply:
> Let me guess, deprecations from cfg.pretty()?

Good guess!